src/tools/spin/spinquic.cpp

```diff
@@ -1062,6 +1147,101 @@ CXPLAT_THREAD_CALLBACK(RunThread, Context)
     CXPLAT_THREAD_RETURN(0);
 }
 
+
+int start(void* Context) {
```

Comment:
Seems this `Context` is never passed in or used? Remove it?

Reply:
Also, you never return anything but `0`? Why have the `int` return then?

Reply:
Removed Context and return void